lib/installer.js

```diff
@@ -225,3 +225,13 @@ function acquireNodeFromFallbackLocation(version) {
         return yield tc.cacheDir(tempDir, 'node', version);
     });
 }
+// os.arch does not always match the relative download url, e.g.
+// os.arch == 'arm' != node-v12.13.1-linux-armv7l.tar.gz
+function translateArchToDistUrl(arch) {
```

Comment:
might be useful to put more detail regarding the arm translation to make it clear the 64 bit version matches but 32 bit doesn't

Reply:
I started using an dictionary lookup in the go setup, which seems more readable and exact to me. Might be worth switching to this pattern so you can just define exceptions:

https://github.com/hross/setup-go/blob/9b5f2fecaf0470def38c1b18a6f299f5d8de94af/src/installer.ts#L91